packages/component-library/src/components/plmg-slider/test/plmg-slider.spec.tsx

```diff
@@ -0,0 +1,18 @@
+// import { newSpecPage } from '@stencil/core/testing';
```

Comment:
![image](https://user-images.githubusercontent.com/14997231/176433339-8fffb35b-21ad-4ddc-8c08-457f1c4f811b.png)

Reply:
![testing_meme13](https://user-images.githubusercontent.com/104433221/176869528-6af02d12-fa9d-4e12-8bc2-9187977a8f85.jpeg)

Reply:
Technically tests now work but they don't really test anything because they only test for the first render not the re-render after we get the container ref. I don't know if there is a way to re-test after the first re-render.